### ISSUE TITLE
Billing Cycles - Email Attachment Error

### ISSUE THREAD

**Issue by kdjstudios** [kdjstudios/SABillingGitlab#1154]
In GitLab by @kdjstudios on Oct 1, 2018, 08:52

Hello Team,

Due to #1141 we have created a new ticket to correctly resolve the underlining issue. Please see what is needed below.

- Need to generate an GUI error when an attached file goes missing on the server; This error need to happen before the emails are sent out.
- We need to know how and why attachment files are going missing in the first place. 
 - We need to establish a fix to prevent files from being removed from the server.

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Jan 29, 2019, 12:40

@amishra @smasih @pchaudhary 

I would just like to confirm that this is the same issue that was addressed in #1153 ? If so please mark this as a secondary and label it as ready to match the primary ticket.

**Comment by kdjstudios** [comment]
In GitLab by @amishra on Jan 29, 2019, 22:20

@kdjstudios 

Thank you for the update.

Yes, you are right this is the same as #1153.

Thanks.

**Comment by kdjstudios** [comment]
In GitLab by @amishra on Jan 29, 2019, 22:21

Marking secondary for  #1153

**Comment by kdjstudios** [comment]
In GitLab by @amishra on Feb 2, 2019, 02:45

@kdjstudios 

We have successfully released on both production server.

Thanks

**Comment by kdjstudios** [comment]
In GitLab by @amishra on Feb 2, 2019, 02:53

assigned to @amishra

**Comment by kdjstudios** [comment]
In GitLab by @kdjstudios on Feb 4, 2019, 10:52

closed
